test/integration/expose-keptn-metric/no-metrics.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: batch/v1
```

Comment:
would it be possible to choose a better name for the file? instead of metric.yaml -> job-existing-metrics.yaml

Same for the other file as well